lib/generators/clearance/routes/templates/routes.rb

```diff
@@ -1,12 +1,12 @@
 resources :passwords, controller: 'clearance/passwords', only: [:create, :new]
-  resource :session, controller: 'clearance/sessions', only: [:create]
+resource :session, controller: 'clearance/sessions', only: [:create]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.